.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -9,8 +9,8 @@ body:
         👉 https://nuxt.com/docs/community/reporting-bugs
 
         Please use a template below to create a minimal reproduction
-        👉 https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz
-        👉 https://codesandbox.io/s/github/nuxt/starter/v3-codesandbox
+        👉 https://stackblitz.com/github/nuxt/starter/tree/v3/
+        👉 https://codesandbox.io/s/github/nuxt/starter/tree/v3/
```

Comment:
if we do this we can likely remove these branches from the starter. I think they exist in case there's anything specific, like https://github.com/nuxt/starter/blob/v3-codesandbox/sandbox.config.json...

Reply:
But then we should also use them for `nuxt.new` (and keep them all up to date)

Reply:
Exactly. I think if both are working well we can just use the v3 branch.